src/api/routes/address.ts

```diff
@@ -165,14 +198,29 @@ export function createAddressRouter(db: DataStore, chainId: ChainID): RouterWith
       return res.status(400).json({ error: `invalid STX address "${stxAddress}"` });
     }
 
+    const until_block = parseUntilBlockQuery(req, res, next);
```

Comment:
Nit: here and a few other areas should use camelCase rather than snake_case (we only use snake_case when dealing with properties from sql queries or http/json requests/responses).